src/terminalLinkProvider.ts

```diff
@@ -29,7 +29,7 @@ export function configureTerminalLinkProvider(
       _token: vscode.CancellationToken
     ): vscode.ProviderResult<TerminalLinkWithData[]> => {
       const regex =
-        /(?:\((?<app>[_a-z]+) \d+.\d+.\d+\) )(?<file>[_a-z/]*[_a-z]+.ex):(?<line>\d+)/;
+        /(?:\((?<app>[_a-z0-9]+) \d+.\d+.\d+\) )(?<file>[_a-z0-9/]*[_a-z0-9]+.ex):(?<line>\d+)/;
```

Comment:
I never noticed that before but those regexes should match unicode file names

Reply:
@lukaszsamson They should!

The other issues to tackle would be all possible `Version` strings, and other popular Elixir extensions.

Have a look at this regex test example I made: https://stackblitz.com/edit/vscode-elixir-ls-regex-test?file=index.js

`newMatch` is the suggested new matcher, `annoMatch` is just to show how it's matching the version
and extension parts.

Run `node index.js` in the integrated terminal.